### PR TITLE
Add 19H2 to bug issue template windows version list

### PR DESCRIPTION
**Pull request by chingucoding**
## Description

Add Windows 10 19H2 to table of windows version in the GitHub bug issue template
## Motivation and Context

Keep bug issue template up to date
## How Has This Been Tested?

Viewed the MD file in GitHub

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ NuGet package version:
 | Windows 10 version                  | Saw the problem? |
 | :--------------------------------- | :-------------------- |
 | Insider Build (xxxxx)              | <!-- Yes/No? -->   |
+| November 2019 Update (18363)       | <!-- Yes/No? -->   |
```
Comment (kaiguo) [inline .github/ISSUE_TEMPLATE/bug_report.md]: The new sdk is probably not published yet, I saw a 19041 preview [here](https://www.microsoft.com/en-us/software-download/windowsinsiderpreviewSDK), not sure if that will be the final version number.

Comment (chingucoding) [reply .github/ISSUE_TEMPLATE/bug_report.md]: According to [this article](https://venturebeat.com/2019/11/12/microsoft-starts-rolling-out-windows-10-november-2019-update/) the build number seems to be 18363, but maybe we should wait till the SDK is released.

Comment (kaiguo) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Thanks for looking it up! @ranjeshj will probably have more info about this.

Comment (Felix-Dev) [reply .github/ISSUE_TEMPLATE/bug_report.md]: 19H2 doesn't come with an own SDK. 20H1, however, will (build 19041 or higher).

**Comment by kaiguo** [comment]
Closing it for now since 19H2 doesn't come with SDK.

**Pull request closed without merging**